props/saber_BC_buttons.h

```diff
@@ -0,0 +1,1092 @@
+      dynamic_mixer.set_volume(std::max<int>(VOLUME * 0.1,
+        dynamic_mixer.get_volume() - VOLUME * 0.10));
+      if (SFX_voldown) {
+        hybrid_font.PlayCommon(&SFX_voldown);
+      } else {
+        beeper.Beep(0.5, 2000);
+      }
+      STDOUT.print("Volume Down - Current Volume: ");
+      STDOUT.println(dynamic_mixer.get_volume());
+    } else {
+      #ifdef VOLUME_MENU_CYCLE
+        if (!min_vol_reached_) {
+          if (SFX_volmin) {
+            hybrid_font.PlayCommon(&SFX_volmin);
+          } else {
+            beeper.Beep(0.5, 1000);
+          }
+          STDOUT.print("Minimum Volume: ");
+          min_vol_reached_ = true;
+        } else {
+          dynamic_mixer.set_volume(VOLUME);
+          if (SFX_volmax) {
+            hybrid_font.PlayCommon(&SFX_volmax);
+          } else {
+            beeper.Beep(0.5, 3000);
+          }
+          STDOUT.print("Maximum Volume: ");
+          min_vol_reached_ = false;
+        }
+      #else
+        if (SFX_volmin) {
+          hybrid_font.PlayCommon(&SFX_volmin);
+        } else {
+          beeper.Beep(0.5, 1000);
+        }
+        STDOUT.print("Minimum Volume: ");
+      #endif
+      
+    }
+  }
+
+
+  bool Event2(enum BUTTON button, EVENT event, uint32_t modifiers) override {
+    switch (EVENTID(button, event, modifiers)) {
+      case EVENTID(BUTTON_POWER, EVENT_PRESSED, MODE_ON):
+      case EVENTID(BUTTON_AUX, EVENT_PRESSED, MODE_ON):
+      case EVENTID(BUTTON_AUX2, EVENT_PRESSED, MODE_ON):
+        if (accel_.x < -0.15) {
+          pointing_down_ = true;
+        } else {
+          pointing_down_ = false;
+        }
+      return true;
+
+
+// Gesture Ignition Controls
+
+  #ifdef BC_SWING_ON
+    case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_OFF):
+      // Due to motion chip startup on boot creating false ignition we delay Swing On at boot for 3000ms
+      if (millis() > 3000) {
+        FastOn();
+  #ifdef GESTURE_AUTO_BATTLE_MODE
+        STDOUT.println("Entering Battle Mode");
+        battle_mode_ = true;
+  #endif
+      }
+      return true;
+  #endif
+
+  #ifdef BC_TWIST_ON
+    case EVENTID(BUTTON_NONE, EVENT_TWIST, MODE_OFF):
+      // Delay twist events to prevent false trigger from over twisting
+      if (millis() - last_twist_ > 2000 &&
+          millis() - saber_off_time_ > 1000) {
+        FastOn();
+  #ifdef GESTURE_AUTO_BATTLE_MODE
+        STDOUT.println("Entering Battle Mode");
+        battle_mode_ = true;
+  #endif
+        last_twist_ = millis();
+      }
+      return true;
+  #endif
+
+  #ifdef BC_TWIST_OFF
+    case EVENTID(BUTTON_NONE, EVENT_TWIST, MODE_ON):
+      // Delay twist events to prevent false trigger from over twisting
+      if (millis() - last_twist_ > 3000) {
+        Off();
+        last_twist_ = millis();
+        saber_off_time_ = millis();
+        battle_mode_ = false;
+      }
+      return true;
+  #endif
+
+  #ifdef BC_STAB_ON
+        case EVENTID(BUTTON_NONE, EVENT_STAB, MODE_OFF):
```

Comment:
this case (and the code that goes with it) seem to be indented differently than the previous case and code

Reply:
Fixed, all identical now.